packages/design/components/Message/style/index.less

```
@@ -0,0 +1,40 @@
+@import '../../../theme/base';
+
+.bgm-message {
+  display: flex;
+
+  &__content {
+    border-radius: 19px;
+    padding: 9px 16px;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+
+    > svg {
+      flex: none;
+      width: 20px;
+      height: 20px;
+      margin-right: 8px;
+    }
+
+    > span {
+      font-weight: 600;
+      font-size: 14px;
+      line-height: 20px;
+    }
+  }
+
+  &--block > &__content {
+    flex: 1;
+  }
```

```suggestion
  &__content--block {
    flex: 1;
  }
```
修饰的 class 应该在 `__content` 上，直接用就行，不需要加 `>`

---

那外面的div就只留个bgm-message，原先的变量类名都放在里面的bgm-message__content；这样可读性的确上升了